modules/library/opengis/pom.xml

```diff
@@ -177,7 +177,7 @@
     
     <dependency>
       <groupId>systems.uom</groupId>
-      <artifactId>systems-common-java8</artifactId>
+      <artifactId>systems-common</artifactId>
```

Comment:
Is there no longer a java 8 specific systems-common?

Reply:
TL;DR They support Java 8 out of the box.

Indriya 2.0 is compiled at source/target level Java 8.  Support for Java8/9+ was one of the reasons they made JSR 385. https://github.com/unitsofmeasurement/indriya/blob/master/pom.xml

It looks like they only support Java 8+:

> Modular environment - While the API should be backward-compatible with at least Java SE 8, both API and RI are expected to expose Java 9 modules
https://www.jcp.org/en/jsr/detail?id=385

Is Java 7 a requirement for GeoTools?

Reply:
We are Java 8 and Java 11 explicitly; so if it is set to run on Java 8 that is fine.